applications/crossbar/src/modules/cb_user_auth.erl

```diff
@@ -132,6 +136,38 @@ put(Context, ?RECOVERY) ->
 %%--------------------------------------------------------------------
 %% @private
 %% @doc
+%% @end
+%%--------------------------------------------------------------------
+-spec maybe_get_auth_token(cb_context:context(), ne_binary()) -> cb_context:context().
+maybe_get_auth_token(Context, Token) ->
+    Context1 = crossbar_doc:load(Token, Context),
+    case cb_context:resp_status(Context1) of
+        'success' ->
+            AuthAccountId = cb_context:auth_account_id(Context),
+            AccountId = cb_context:account_id(Context),
+            create_auth_resp(Context1, AccountId, Token, AuthAccountId);
```

Comment:
Why switch up the argument order here? Why not `create_auth_resp(Context1, Token, AccountId, AuthAccountId)`, since you came into the function with `(Context, Token)`?